src/inspector_io.cc

```diff
@@ -182,8 +182,11 @@ bool InspectorIo::Start() {
 }
 
 void InspectorIo::Stop() {
+  if (State::kNew == state_)
```

Comment:
Style nit: it's not really wrong but can you follow the dominant non-yoda style?